ci/Jenkinsfile.release

```
@@ -9,96 +9,27 @@ def wait_for_db() {
     }
 }
 
-stage('Build Schema') {
-    node('docker') {
-        node_info()
-
-        checkout scm
-        def rustImage = docker.build('permaplant-rust', './ci/container-images/permaplant-rust')
-
-        docker.image('postgis/postgis:13-3.1').withRun('-e "POSTGRES_USER=ci" -e "POSTGRES_PASSWORD=ci"') { c ->
-            rustImage.inside("--link ${c.id}:db -e 'DATABASE_URL=postgres://ci:ci@db/ci'") {
-                checkout scm
-
-                wait_for_db()
-
-                sh './ci/build-scripts/build-schema.sh'
+// Imports
+node('docker') {
+    checkout scm
+    common = load('ci/Jenkinsfile.groovy')
+}
 
-                stash includes: 'backend/src/schema.rs', name: 'schema.rs'
-                stash includes: 'frontend/src/api_types/definitions.ts', name: 'definitions.ts'
-            }
-        }
-    }
+stage('Build Schema') {
+    common.runDockerPostgresSidecar('echo schema test', schemaStashsrc, schemaStashdir)
```

What is `schema test`? Shouldn't it be:

```suggestion
    common.runDockerPostgresSidecar('echo Build Schema', schemaStashsrc, schemaStashdir)
```

---

this is just a meaningless echo, so the function works.

---

Yeah, but "test" gives the impression this would test something but this isn't the case? So either no text or text that is correct.

---

The function `runDockerPostgresSidecar` always builds the schemas, so its basically a dummy call but named `schema test` because that is its purpose.